auto_round/compressors/base.py

```diff
@@ -336,7 +337,11 @@ def __init__(
         self.lr_scheduler = lr_scheduler
         self.optimizer = self._get_optimizer(None)
         self.disable_opt_rtn = disable_opt_rtn
-        self.is_packing_immediate = False  # whether to pack the layer immediately after tuning
+
+        # Whether to pack the layer immediately after tuning
+        self.immediate_packing = False
+        self.immediate_saving = False
+        self.low_cpu_mem_usage = kwargs.pop("low_cpu_mem_usage", False)
```

Comment:
move the code in front of the following, otherwise, there is a warning
~~~python
        if kwargs:
            logger.warning(f"unrecognized keys {list(kwargs.keys())} were passed. Please check them.")
~~~